CONTRIBUTING.rst

```diff
@@ -55,28 +55,33 @@ Ready to contribute? Here's how to set up CumulusCI for local development.
 
 1. Fork the CumulusCI repo on GitHub.
 2. Clone your fork to your local workspace.
-3. Create a fresh virtual environment using virtualenv and install development requirements::
+3. Create a fresh Python virtual environment and activate it (to keep this isolated from other Python software on your machine). Here is one way::
+
+    $ python -m venv cci_venv
```

Comment:
We might need some more detail here. On macOS, `python` yields Python 2.7.

Reply:
Good point, probably best to use `python3`